bitwarden_license/src/Sso/Controllers/AccountController.cs

```diff
@@ -209,6 +209,8 @@ public IActionResult ExternalChallenge(string scheme, string returnUrl, string s
             returnUrl = "~/";
         }
 
+        // Clean the returnUrl of any whitespace characters
+        returnUrl = returnUrl.Replace(" ", string.Empty).Replace("\t", string.Empty);
```

Comment:
I did some quick sleuthing, but nothing conclusive on whether or not "\r" or "\n" also need to be addressed?

Reply:
🎨 To pile on a bit, I think it's a good idea to put this in a utility method somewhere instead of just here.

Few approaches here: https://stackoverflow.com/questions/6219454/efficient-way-to-remove-all-whitespace-from-string